### PR TITLE
Add 'install' keyword to the gettext macro

### PR DESCRIPTION
**Pull request by tintou**

### CODE REVIEW

**Comment by tintou** [comment]
Example of use:
I have a gettext domain on my project that is only meant to be merged (with i18n.merge_file) on .appdata.xml and .desktop so I don't want it to be installed on the target system.

File: mesonbuild/modules/i18n.py [hunk 1]
```diff
@@ -126,16 +126,21 @@ def gettext(self, state, args, kwargs):
...
-                '--localedir=' + state.environment.coredata.get_builtin_option('localedir'),
-                pkg_arg]
-        if lang_arg:
-            args.append(lang_arg)
-        iscript = build.RunScript(script, args)
-
-        return ModuleReturnValue(None, [pottarget, gmotarget, iscript, updatepotarget])
+        targets = [pottarget, gmotarget, updatepotarget]
+
+        install = kwargs.get('install') if ('install' in kwargs) else True
```
Comment (TingPing) [inline mesonbuild/modules/i18n.py]: `install = kwargs.get('install', True)`

**Comment by tintou** [comment]
@TingPing changes are made :)

**Comment by jpakkane** [comment]
Please update the corresponding module documentation in `docs/markdown`.

**Comment by tintou** [comment]
@jpakkane done

**Pull request merged**